src/CMakeLists.txt

```diff
@@ -1,7 +1,18 @@
 
 include(SwiftSupport)
 
+set(dispatch_DTRACE_HEADERS)
+if(dtrace_EXECUTABLE)
+  list(APPEND dispatch_DTRACE_HEADERS ${CMAKE_CURRENT_BINARY_DIR}/provider.h)
+  ADD_CUSTOM_COMMAND(
```

Comment:
Lower case please.  Can you please create a target for this?  This will otherwise run constantly.  I can quickly write a patch to prevent that if you like.